src/fitnesse/testsystems/slim/SlimTestContextImpl.java

```diff
@@ -14,6 +15,15 @@
   private final Map<String, String> symbols = new HashMap<String, String>();
   private final Map<String, ScenarioTable> scenarios = new HashMap<String, ScenarioTable>();
   private final TestSummary testSummary = new TestSummary();
+  private TestPage pageToTest;
+
+  public SlimTestContextImpl(TestPage pageToTest) {
+    this.pageToTest = pageToTest;
+  }
+  
+  public SlimTestContextImpl() {
+    super();
+  }
```

Comment:
I think the default constructor can go. It should no longer be needed, right?

cc: @fhoeben